core-bundle/src/Twig/Extension/ImageExtension.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Twig\Extension;
+
+use Contao\CoreBundle\Twig\Runtime\FigureRendererRuntime;
+use Contao\CoreBundle\Twig\Runtime\PictureConfigurationRuntime;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class ImageExtension extends AbstractExtension
+{
+    public function getFunctions(): array
+    {
+        return [
+            new TwigFunction(
+                'contao_figure',
+                [FigureRendererRuntime::class, 'render'],
+                ['is_safe' => ['html']]
+            ),
+            new TwigFunction(
+                'picture_configuration',
+                [PictureConfigurationRuntime::class, 'fromArray']
+            ),
+        ];
```

Comment:
In case you're wondering "Why the Runtime pattern?": 

1) It will only instantiate the runtime when the function is used in a rendered template (vs. everytime).
   see https://symfony.com/doc/current/templating/twig_extension.html#creating-lazy-loaded-twig-extensions
2) It's easier to register the functions under another name if you ever need to.